docs/GETTING-STARTED.md

```diff
@@ -50,17 +31,21 @@ or using script which opens defualt proxy (the same as above one)
 ```bash
 ssh -D localhost:12345 <user>@<host>
 ```
-then save the address of the socket you've opened in CERN_SOCKS env var e.g. 
+then save the address of the socket you've opened in `CERN_SOCKS` env var e.g. 
```

Comment:
The address of the socket you've opened should be saved in `CERN_SOCKS` env var, e.g.: